save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -31,6 +29,7 @@ typealias VulnerabilityDtoList = List<VulnerabilityDto>
 @RequestMapping(path = ["/api/$v1/vulnerabilities"])
 class VulnerabilityController(
     private val vulnerabilityService: VulnerabilityService,
+    private val vulnerabilityDescriptionService: VulnerabilityDescriptionService,
```

Comment:
Why do you need `VulnerabilityDescriptionService`? I mean `VulnerabilityDescription` is just a pair of `Vulnerability` id and its description. Maybe encapsulate all the `Vulnerability` logic in `VulnerabilityService`

Reply:
Agree. Deleted _VulnerabilityDescriptionService_